regtest/data/zcashd/regtest/blocks/index/LOG

```diff
@@ -0,0 +1,5 @@
+2022/08/26-00:13:45.135344 7f00d4cece80 Recovering log #6
```

Comment:
I suspect we want to `.gitignore` the LOG files, and definitely the LOCK/.lock files.

Reply:
I expected them to be a problem, but they don't seem to be. It's working locally.